### DISCUSSION TITLE
meaning of blesk?

### DISCUSSION THREAD

**Discussion by defdefred** [VendelinSlezak/BleskOS#75] [Q&A] [answered]
Could you explain what "blesk" mean?
Is it "lightning" to express the speed of the code?

**Comment by VendelinSlezak** [comment, marked as answer]
Yes, exactly. "blesk" is "lightning" in slovak and it refers to speed of BleskOS compared to classis OSes. In fact, original name of this project was LightningOS, but then I found that there already is one linux distro with this name, so I renamed it with word from my native language. BTW Did you use translator, or do you know this word by some other way?

**Reply by defdefred** [reply]
translator...
